.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug Report
+about: No one likes bugs
+title: "[BUG] "
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+A clear and concise description of what the bug is.
+
+## Does this bug happen in the Forge version?
+Tell us whether this bug is specific to the Fabric version.
+
+## To Reproduce
+Steps to reproduce the behavior:
+1. Go to '...'
+1. Click on '....'
+1. Scroll down to '....'
+1. See error
+
+## Expected behavior
+A clear and concise description of what you expected to happen (e.g. description of what happens in the Forge version if the bug is only here).
+
+## Screenshots
+If applicable, add screenshots to help explain your problem.
+
+## System information (please complete the following information):
```

Comment:
I think non-JiJed dependency versions and a list of other mods should be provided as well. Someone could report that connected textures don't work with Sodium, but that is not Create's bug.